data_summaries/MathInstruct.json

```diff
@@ -0,0 +1,75 @@
+{
+    "mathinstruct": {
+        "Unique Dataset Identifier": "mathinstruct",
+        "Dataset Name": "mathinstruct",
+        "Paper Title": "MAmmoTH: Building Math Generalist Models through Hybrid Instruction Tuning",
+        "Dataset URL": "https://huggingface.co/datasets/TIGER-Lab/MathInstruct",
+        "GitHub URL": "https://github.com/TIGER-AI-Lab/MAmmoTH",
+        "Hugging Face URL": "https://huggingface.co/datasets/TIGER-Lab/MathInstruct",
+        "Papers with Code URL": "",
+        "ArXiv URL": "https://arxiv.org/abs/2309.05653",
+        "Semantic Scholar Corpus ID": 261696697,
+        "Collection": "MathInstruct",
+        "Collection URL": "https://github.com/TIGER-AI-Lab/MAmmoTH",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Chain-of-Thought",
+            "Instruction Following",
+            "Logical Reasoning",
+            "Logical Reasoning Question Answering",
+            "Probabilistic Reasoning",
+            "Probabilistic Reasoning Question Answering",
+            "Algebraic Expression Evaluation"
+        ],
+        "Text Sources": [],
+        "Model Generated": [
+            "OpenAI GPT-4"
+        ],
+        "Format": [
+            "Zero-shot",
+            "Chain-of-Thought",
+            "Program-of-Thought"
+        ],
+        "Human Annotation": "No",
+        "Derived from Datasets": [
+            "GSM8K",
+            "GSM8K-RFT",
+            "AQuA-RAT",
+            "MATH",
+            "TheoremQA",
+            "Camel-Math",
+            "NumGLUE",
+            "MathQA"
+        ],
+        "Creators": [
+            "University of Waterloo",
+            "The Ohio State University"
+        ],
+        "Licenses": [
+            {
+                "License": "MIT License",
+                "License URL": "https://github.com/TIGER-AI-Lab/MAmmoTH#license"
+            }
```

Comment:
Add OpenAI License here in addition to the GitHub license.

Reply:
Done